docs/components/buttons.md

```diff
@@ -1,38 +1,190 @@
 
-This is some text about link buttons. We should explain why we want links to
-look like buttons.
+## When to use
+
+Use a button when an action has more weight or significance. Examples:
+
+* There is a clear primary action people should take
+* Highlighting a suggested action in a page section
+* A significant or destructive action will be applied and users need to be aware
+
+If the action should not be prompted, a text link may be a better option.
+
+## Types of buttons
+
+### Primary button
+
+This is the most common type of button. Use these to direct people to a clear primary action. To call out that these buttons are actionable, they are action blue (`{{ tokens.colors.action }}`).
+
+```html
+<button class="btn">
+  Do something
+</button>
+```
+
+Try not to have more than 1 primary button on a page. Multiple primary buttons puts more cognitive load on people. It takes them more time and effort to figure out what to do.
+
+### Inverse button
+
+On some backgrounds, the primary button color will not have sufficient contrast. In this case, use inverse buttons, which have a white fill.
+
+```html wrapper_class="bg-blue-dark p-20"
+<div class="bg-blue-dark">
+  <button class="btn btn-inverse">
+    Do something
+  </button>
+</div>
+```
+
+### Secondary button
+
+```html
+<button class="btn btn-secondary">
+  Do something
+</button>
+```
+
+Secondary buttons don’t have as much visual weight because they are outlined instead of solid. Use these if your button is not an important action on the page. 
+
+Pair it side-by-side with a primary button to prompt toward the primary button’s action.
+
+```html id="buttons-side-by-side"
+<div class="flex gap-20">
+  <button class="btn btn-secondary">
+    Cancel
+  </button>
+  <button class="btn">
+    Continue
+  </button>
+</div>
+```
+
+## Variations
+
+### Block buttons
+
+Block buttons stretch to fill the width of the screen or area instead of having a set width. They are often used on mobile layouts.
+
+```html
+<div class="bg-blue-1 p-20 text-slate text-body">
+  <div>COVID-19 vaccination appointments available only for San Francisco Health Network patients.</div>
+  <button class="btn btn-block my-20">
+    Check for availability
+  </button>
+  <div class="text-center">
+    Or call <a href="tel:1234567890" class="text-action">123-456-7890</a>
+  </div>
+</div>
+```
+
+### Icons
+
+Icons can be on the left or right of the text in a button. Do not use more than one icon in a button.
+
+```html
+<div class="flex justify-around items-center gap-16 title-xs">
+  <a href="#" class="btn btn-secondary flex gap-8">
+    <sfgov-icon symbol="arrow-left"></sfgov-icon>
+    <span>Previous</span>
+  </a>
+  <a href="#" class="text-action">1</a>
+  <span>...</span>
+  <a href="#" class="text-action">8</a>
+  <a href="#" class="text-action">9</a>
+  <span>10</span>
+  <a href="#" class="text-action">11</a>
+  <a href="#" class="btn btn-secondary flex gap-8">
```

Comment:
clicking these affects scroll position in a kinda jarring way; I believe doing the following will fix the page jump
```suggestion
  <a href="#/" class="btn btn-secondary flex gap-8">
    <sfgov-icon symbol="arrow-left"></sfgov-icon>
    <span>Previous</span>
  </a>
  <a href="#/" class="text-action">1</a>
  <span>...</span>
  <a href="#/" class="text-action">8</a>
  <a href="#/" class="text-action">9</a>
  <span>10</span>
  <a href="#/" class="text-action">11</a>
  <a href="#/" class="btn btn-secondary flex gap-8">
```